agent/config/runtime.go

```diff
@@ -22,6 +22,17 @@ type RuntimeSOAConfig struct {
 	Minttl  uint32 // 0,
 }
 
+// CacheRateLimitPerEntry is a representation with human-readable value
+type CacheRateLimitPerEntry struct {
+	Value           string
+	RateLimitConfig *lib.RateLimitSpec
+}
+
+// CacheConfiguration represent the cache configuration of agent
+type CacheConfiguration struct {
+	CacheRateLimitPerEntry CacheRateLimitPerEntry
```

Comment:
I think `Cache` is implied by the name of the struct.  It may also not be obvious what is being rate limited. How about something like `EntryFetchRateLimit` ?

Reply:
ok, I'll change that